### PR TITLE
OCPBUGS-16170: Add support for Intel X710 Backplane and Base T

### PR DESCRIPTION
**Pull request by SalDaniele**
Backport https://github.com/openshift/sriov-network-operator/pull/794 to 4.13

### CODE REVIEW

**Comment by SalDaniele** [comment]
@zhaozhanqi @wizhaoredhat PTAL, need approvals to backport https://github.com/openshift/sriov-network-operator/pull/794 to 4.13

File: doc/supported-hardware.md [hunk 1]
```diff
@@ -54,6 +61,7 @@ The following table depicts the supported SR-IOV hardware features of each suppo
 | Mellanox MT28908 Family [ConnectX-6 Lx] | V | V | V |
 | Mellanox MT28908 Family [ConnectX-7] | V | V | V |
 | Mellanox MT42822 BlueField-2 integrated ConnectX-6 Dx | V | V | V |
+| Mellanox MT43244 BlueField-3 integrated ConnectX-6 Dx | V | V | V |
```
Comment (wizhaoredhat) [inline doc/supported-hardware.md:64]: Did we accidentally pull in another commit?

Comment (SalDaniele) [reply doc/supported-hardware.md:64]: Yeah I was a bit confused by this inclusion. Not sure if this was intentional. I will remove this for now, if we need to backport cx-7 support we can do this in a separate PR

**Comment by wizhaoredhat** [comment]
@zhaozhanqi Could you provide your approval?

**Comment by zhaozhanqi** [comment]
need label `backport-risk-assessed` firstly,  and then I can  /label cherry-pick-approved

**Comment by zhaozhanqi** [comment]
> need label `backport-risk-assessed` firstly, and then I can /label cherry-pick-approved

sounds like `backport-risk-assessed` is failed to add :)

**Comment by wizhaoredhat** [comment]
> > need label `backport-risk-assessed` firstly, and then I can /label cherry-pick-approved
> 
> sounds like `backport-risk-assessed` is failed to add :)

Should be good now.
